openmole/plugins/org.openmole.plugin.task.netlogo/src/main/scala/org/openmole/plugin/task/netlogo/NetLogoTask.scala

```diff
@@ -148,11 +172,21 @@ trait NetLogoTask extends Task with ValidateTask {
 
       for (inBinding ← netLogoInputs) {
         val v = preparedContext(inBinding._1) match {
-          case x: String ⇒ '"' + x + '"'
-          case x: File   ⇒ '"' + x.getAbsolutePath + '"'
-          case x         ⇒ x.toString
+          case x: File   ⇒ x.getAbsolutePath
+          case x: AnyRef ⇒ x
+        }
+        NetLogoTask.setGlobal(instance.netLogo, inBinding._2, v)
+      }
+
+      for (inBindingArrays ← netLogoArrayInputs) {
```

Comment:
I think the test whether the prototype is an array is sufficient. There is no need for a distinct setting.